styles/Home.module.css

```diff
@@ -287,3 +287,40 @@ input:focus {
 #sub:hover > div > div {
   background: rgb(159, 154, 255) !important;
 }
+
+
+
+
+
+/* Styles for the container wrapping the Quotes component */
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh; /* Adjust this value based on your layout */
+}
```

Comment:
Which component does this style apply to ? If this is any inner specific component I would suggest to give a unique name since there are chances that a `.container` must be used elsewhere in codebase and this would create confusions going ahead.

Reply:
it's a default component